### PR TITLE
Increase max heap of LS simulator

### PR DESCRIPTION
**Pull request by IMS94**
## Purpose
$subject and add 2201.1.1-stage branch

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: .github/workflows/language_server_simulator.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix: 
-        branch: ["master", "2201.0.x", "2201.1.x", "2201.1.0-stage"]
+        branch: ["master", "2201.0.x", "2201.1.x", "2201.1.0-stage", "2201.1.1-stage"]
```
Comment (malinthar) [inline .github/workflows/language_server_simulator.yml:16]: Shall we remove 2201.1.0-stage

**Review by malinthar — DISMISSED** [review]
LGTM!

**Review by mohanvive — APPROVED** [review]

File: .github/workflows/language_server_simulator.yml (hunk 1 above)
Comment (IMS94) [reply .github/workflows/language_server_simulator.yml]: Done!

**Review by malinthar — APPROVED** [review]
LGTM!

**Pull request merged**